invenio_records_rest/views.py

```diff
@@ -69,6 +68,12 @@ def create_blueprint(endpoints):
         for rule in create_url_rules(endpoint, **options):
             blueprint.add_url_rule(**rule)
 
+    # catch record validation errors
+    @blueprint.errorhandler(ValidationError)
+    def validation_error(error):
+        """Catch validation errors."""
+        return RESTException(description="Validation error"), 400
```

Comment:
What about a generic ``RESTValidatinoError`` in that models responses according to  https://zenodo.org/dev#restapi-errors. I.e.:

```json
{
    "message": "Validation error",
    "status": 400,
    "errors": [
        {"code": 10, "message":"Not a valid choice", "field": "access_right"}
    ]
}
```

See https://github.com/inveniosoftware/invenio-rest/pull/34/files

```python
from invenio_rest.errors import RESTValidationError, FieldError

raise RESTValidationError(errors=[FieldError('access_right', 'Not a valid choice', code=10)])
```

The RESTValidationError will be caught by Invenio-REST error handlers.